v2/account_claims.go

```diff
@@ -72,26 +71,36 @@ func (j *JetStreamLimits) IsJSEnabled() bool {
 	return j.MemoryStorage != 0 || j.DiskStorage != 0
 }
 
+type TieredLimits map[string]JetStreamLimits
+
 // OperatorLimits are used to limit access by an account
 type OperatorLimits struct {
 	NatsLimits
 	AccountLimits
 	JetStreamLimits
+	TieredLimits `json:"tiered_limits,omitempty"`
 }
 
-// IsEmpty returns true if all of the limits are 0/false.
+// IsEmpty returns true if all limits are 0/false/empty.
 func (o *OperatorLimits) IsEmpty() bool {
-	return *o == OperatorLimits{}
+	return o.NatsLimits == NatsLimits{} &&
```

Comment:
Wonder if you could replace all that with: `reflect.DeepEqual(o, &OperatorLimits{})`? (would need to be tested, but..)

Reply:
if my read of deep equal is correct, a nil map and an empty map would return false, where I don't really care.
Someone initializing their own struct could run into this: https://play.golang.com/p/6zz-J_UYVOt
This prints `false`
```
	var v1 map[string]string
	v2 := map[string]string{}
	fmt.Println(reflect.DeepEqual(v1, v2))
```

Reply:
I see.. yes, `nil` is different that initialized map with no entry.